### DISCUSSION TITLE
Jupyter Notebook crashes when skimming code with WSL2

### DISCUSSION THREAD

**Discussion by dibbla** [microsoft/vscode-jupyter#10177] [Questions and Answers]
Hi guys，I'm now using WSL2Ubuntu with VSC to develop. However, my computer crashes several time when I was just skmming the code without running anything. I check the monitor and it reports that the memory is 100%. 

Here is my system info:
Win11 21H2
Intel i7 12700H
Nvidia RTX3060 laptop
16GB memory

VSC Jupyter Extension v2022.4.1021342353
VSC 1.67.2

Is it a problem with WSL2 or VSC server or both? ANY idea?

**Comment by dibbla** [comment]
> Hi guys，I'm now using WSL2Ubuntu with VSC to develop. However, my computer crashes several time when I was just skmming the code without running anything. I check the monitor and it reports that the memory is 100%.
> 
> Here is my system info: Win11 21H2 Intel i7 12700H Nvidia RTX3060 laptop 16GB memory
> 
> VSC Jupyter Extension v2022.4.1021342353 VSC 1.67.2
> 
> Is it a problem with WSL2 or VSC server or both? ANY idea?

More Info:
The code is from [here](https://colab.research.google.com/drive/1FTcG6CE-HILnvFztEFKdauMlPKfQvm5Z#scrollTo=2jc7ZfDot2t9)

**Reply by DonJayamanne** [reply]
If you aren't running any code in the cells and it still crashes then its likely caused by some other extension.
Please could you disable the Juptyer extension and confirm it still reports a 100% memory usage. 
Its possible this is caused by the python or some other related/similar extension.
